### PR TITLE
Updating Alias tutorials to Drac2

### PR DESCRIPTION
**Pull request by MobiusMage**
### Summary
Updating the alias tutorials on the website to drac2. The first (orc-relentless) required more updating to replace deprecated methods with the character( ) equivalents. The second (insult) is mostly intact but swapped the {{ }} syntax for a <drac2></drac2> code block to demonstrate other options for beginning aliasers. orc-relentless was left witht he {{ }} syntax to not overcomplicate it with indentation.

#### PR Type

- [ ] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [ ] This PR adds a new feature that is not an open feature request.
- [x] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] If code changes were made then they have been tested.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

**Review by zhudotexe — APPROVED** [review]
LGTM! Extra credit: convert the Relentless Endurance alias to drac2 (it looks like a great candidate to show off if-statements in drac2).

**Comment by MobiusMage** [comment]
> LGTM! Extra credit: convert the Relentless Endurance alias to drac2 (it looks like a great candidate to show off if-statements in drac2).

I originally rewrote it with if: blocks and reconsidered since I wasn't sure how well the indents would show up in the line-by-line explanation of the code and was concerned it might be confusing to people just starting out in aliasing. I should still have that code, though, and could write up another revision to see how it looks.

File: docs/aliasing/aliasing_tut.rst [hunk 1]
```diff
@@ -23,68 +23,175 @@ This is the base Avrae command, an embed, which makes the pretty text box. Check
...
+``ch.create_cc_nx`` This will create a custom counter on the character (``ch``) if it doesn't already exist. 
+``cc`` This defines the name of the counter. In this case, it uses one of the variables declared earlier, so the counter will be ``Relentless Endurance``
+``0, 1`` The next two arguments define the minimum and maximum values of the CC. Since this can only ever be used once at a time, this counter can only go between 0 (used) and 1 (available)
+``"long"`` Next we define how the counter resets. We're specifying that it should reset on a Long Rest.
+``"bubble"`` This specifies how the counter should be presented. Bubble gives a depiction of the counter that is more visual and aesthetically pleasing
+``None, None`` These next two are Reset To and Reset By, respectively. They are optional arguments for more advanced custom counters, and aren't needed for this one.
+``cc`` The next argumet is the Title of the counter that will be seen when setting or viewing the counter. We're just setting it to the same thing as its title.
+``desc+" "+rest`` Finally, this is the counter's description. We're using two of the previously-defined variables, joined with a space between them.
```
Comment (zhudotexe) [inline docs/aliasing/aliasing_tut.rst:71]: ReST treats single line breaks as a space - consider using two line breaks or a bulleted list

Comment (zhudotexe) [reply docs/aliasing/aliasing_tut.rst:71]: Screenshot 2021-04-01 at 10 03 26

File: docs/aliasing/aliasing_tut.rst [hunk 2]
```diff
@@ -23,68 +23,175 @@ This is the base Avrae command, an embed, which makes the pretty text box. Check
...
+``ch.get_cc(cc)`` This gets the value of the counter, which will be 0 (used) or 1 (not used). If-checks treat zero as False, and non-zero as True. So, if the counter is used, the if-check will fail here.
+``not ch.hp`` Checks the character's hp. As before, zero hit points will be considered False, and non-zero is True. The ``not`` before hand will reverse that. That means that if the character has any HP left, the if-check will fail.
+
+If all the conditions are met, the alias will execute the code inside the block. Note that each of these lines has the same indentation. This block will do most of the mechanics work the alias is meant for. Going line-by-line:
+
+``succ = "uses"`` This is the success case that will override this variable to indicate a successful use instead of a failed attempt.
+``D = desc`` This just sets one variable to another. The alias will use ``D`` later when showing the result to the player
+``ch.mod_cc(cc, -1)`` This will modify (``mod_cc``) the value of the counter (``cc``) by ``-1``, reducing it from 1 to 0 and marking it as used
+``ch.set_hp(1)`` This sets the character's hitpoints to 1.
```
Comment (zhudotexe) [inline docs/aliasing/aliasing_tut.rst:101]: See above

**Pull request merged**